### PR TITLE
1185 AM update file

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Checklist
Reminder: If any changes have been released in the [census-rm-sample-loader](https://github.com/ONSdigital/census-rm-sample-loader) or [census-rm-toolbox](https://github.com/ONSdigital/census-rm-toolbox/) then the version pins in the Pipfile need to be updated.
* [x] Updated census-rm-toolbox version pin (if applicable)
* [x] Updated census-rm-sample-loader version pin (if applicable)

# Motivation and Context
Tests or the address management update file bulk processing and internal events.

# What has changed
* Add scenario for bulk address updates
* Update PR template with version bumping reminders

# How to test?
Run against linked service branches

# Links
https://trello.com/c/u0QkoepB/1185-am-update-file-21
https://github.com/ONSdigital/census-rm-docker-dev/pull/68
https://github.com/ONSdigital/census-rm-kubernetes/pull/315
https://github.com/ONSdigital/census-rm-case-processor/pull/201

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ python_version = "3.7"
 
 [packages]
 census-rm-sample-loader = {editable = true,git = "https://github.com/ONSdigital/census-rm-sample-loader",ref = "v2.1.0"}
-census-rm-toolbox = {editable = true,git = "https://github.com/ONSdigital/census-rm-toolbox",ref = "v6.0.0"}
+census-rm-toolbox = {editable = true,git = "https://github.com/ONSdigital/census-rm-toolbox",ref = "b3e191fe1200cac84fefc56e87a83aeaf25bc912"} # TODO update with release tag
```
Comment (AdamHawtin) [inline Pipfile]: This will be updated with the toolbox version once that PR is merged

**Comment by NickGrantX** [comment]
[AM- Update file (21)](https://trello.com/c/u0QkoepB/1185-am-update-file-21)

File: acceptance_tests/features/steps/bulk_processing.py [hunk 2]
```diff
@@ -202,6 +203,116 @@ def check_address_invalid_case_updated_events(context):
...
+    if Config.BULK_ADDRESS_UPDATE_BUCKET_NAME:
+        BulkProcessor(AddressUpdateProcessor()).run()
+        return
+
+    # If we don't have a bucket, mock the storage bucket client interactions to work with only local files
+    with mock_bulk_processor_bucket(context.bulk_address_updates_file):
+        BulkProcessor(AddressUpdateProcessor()).run()
+
+
+@step('CASE_UPDATED events are emitted for all the updated cases with correctly updated data and skeleton marker false')
```
Comment (NickGrantX) [inline acceptance_tests/features/steps/bulk_processing.py:272]: I've seen some ATs (like the one for type change) check the DB too using the Case API

File: config.py [hunk 3]
```diff
@@ -127,3 +127,27 @@ class Config:
     BULK_REFUSAL_BUCKET_NAME = os.getenv('BULK_REFUSAL_BUCKET_NAME')
     BULK_NEW_ADDRESS_BUCKET_NAME = os.getenv('BULK_NEW_ADDRESS_BUCKET_NAME')
     BULK_INVALID_ADDRESS_BUCKET_NAME = os.getenv('BULK_INVALID_ADDRESS_BUCKET_NAME')
+
+    BULK_ADDRESS_UPDATE_FILE_PREFIX = os.getenv('BULK_ADDRESS_UPDATE_FILE_PREFIX', 'address_updates_')
+    BULK_ADDRESS_UPDATE_BUCKET_NAME = os.getenv('BULK_ADDRESS_UPDATE_BUCKET_NAME')
+    BULK_ADDRESS_UPDATE_PROJECT_ID = os.getenv('BULK_ADDRESS_UPDATE_PROJECT_ID')
+    BULK_ADDRESS_UPDATE_ROUTING_KEY = os.getenv('BULK_ADDRESS_UPDATE_ROUTING_KEY', 'case.rm.updated')
+
+    EVENTS_EXCHANGE = os.getenv('EVENTS_EXCHANGE', 'events')
```
Comment (liamtoozer) [inline config.py]: Don't think this is used?

**Review by liamtoozer — APPROVED** [review]

**Review by ryangrundy7 — APPROVED** [review]
Tests passed for me 👍

**Review by NickGrantX — APPROVED** [review]
Marvellous

**Pull request merged**